src/server/test/db/loadFromCsvStreamTests.js

```diff
@@ -90,10 +90,12 @@ mocha.describe('Read mamc log from a file: ', () => {
 				const endTimestamp = moment(row[1], 'MM/DD/YYYY HH:mm');
 				const startTimestamp = moment(row[1], 'MM/DD/YYYY HH:mm').subtract(60, 'minutes');
 				return new Reading(meter.id, readRate, startTimestamp, endTimestamp);
-			}, (readings, tx) => Reading.insertOrIgnoreAll(readings, tx));
+			}, (readings, tx) => Reading.insertOrIgnoreAll(readings, tx),
+			conn
```

Comment:
Again, I'm no expert on this logic but want to check conn is set here and below.